lib/tasks/envirotech.rake

```diff
@@ -0,0 +1,10 @@
+namespace :envirotech do
+  desc 'Creates a dev admin user to access the api'
+  task :lookup_issue, [:issue_name] => :environment do |_t, args|
+    puts Envirotech::ToolkitScraper.new.fetch_issue_info(args.issue_name)
+  end
+
+  task all_issue_info: :environment do
+    puts Envirotech::ToolkitScraper.new.all_issue_info.to_json
+  end
```

Comment:
Do you think it would be better to have this task write directly to data/envirotech/issue_solution_regulation.json? I don't see too much use in writing this big blog of JSON to stdout.